fastlane/actions/verify_keys.rb

```diff
@@ -0,0 +1,42 @@
+module Fastlane
+  module Actions
+
+    class VerifyKeysAction < Action
+      def self.run(params)
+
+        # Ensure no deploys with OSS keys
+        require 'cocoapods-core'
+        puts "Validating CocoaPods Keys"
+
+        podfile = Pod::Podfile.from_file "Podfile"
```

Comment:
@ashfurrow Basically just forking the fastlane repo, adding the action to `actions` folder and adding documentation to the action and to `Actions.md` does the trick :+1: